incubator/sparkoperator/README.md

```diff
@@ -24,7 +24,7 @@ The following table lists the configurable parameters of the Spark operator char
 | Parameter                 | Description                                                  | Default                                |
 | ------------------------- | ------------------------------------------------------------ | -------------------------------------- |
 | `operatorImageName`       | The name of the operator image                               | `gcr.io/spark-operator/spark-operator` |
-| `operatorVersion`         | The version of the operator to install                       | `v2.4.0-v1alpha1-latest`               |
+| `operatorVersion`         | The version of the operator to install                       | `v2.4.0-v1beta1-latest`               |
 | `imagePullPolicy`         | Docker image pull policy                                     | `IfNotPresent`                         |
 | `sparkJobNamespace`       | K8s namespace where Spark jobs are to be deployed            | `default`                              |
```

Comment:
Can we change this from `default` to an empty string, which is the default in https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/main.go#L58?

Reply:
Ok. In `spark-rbac.yaml`, I fixed the condition. When sparkJobNamespace is empty string, the file is not applied.